README.md

```diff
@@ -10,11 +10,21 @@ Model registry provides a central repository for model developers to store and m
 
 ## Pre-requisites:
 - go >= 1.21
+- go-enum >= 1.2.97
+- protoc-gen-go >= 1.31.0
+- protoc-gen-go-grpc >= 1.3.0
+- goverter >= 1.4.1
 - protoc v24.3 - [Protocol Buffers v24.3 Release](https://github.com/protocolbuffers/protobuf/releases/tag/v24.3)
 - npm >= 10.2.0 - [Installing Node.js and npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
 - Java >= 11.0
 - python 3.9
 
+To install the go dependencies you can use
+
+```sh
+echo golang.org/dl/go1.21.9@latest github.com/searKing/golang/tools/go-enum@v1.2.97 google.golang.org/protobuf/cmd/protoc-gen-go@v1.31.0 google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.3.0 github.com/jmattheis/goverter/cmd/goverter@v1.4.1 | xargs -n1 go install
+```
+
```

Comment:
see above related comment; I'd rather have this taken care somehow by the makefile invocation.

I see some project further automate this for GoLang by having toolchains listed in go.mod too, but didn't dig deeper.